src/test/java/com/ahsoka/SALC/user_model/controller/EmailValidatorTest.java

```diff
@@ -0,0 +1,19 @@
+package com.ahsoka.SALC.user_model.controller;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
+class EmailValidatorTest {
+
+    @Autowired
+    EmailValidator emailValidator;
+
+    @Test
+    void testEmailValidator() {
```

Comment:
Prueba a hacer test parametrizados, por ejemplo: 
```suggestion
    
    @ParameterizedTest
    @ValueSource(strings = {"test_admin@salc.org", "test@salc.org", "userEmail@salc.org", "an.user@salc.org", "another-user@salc.org", "user1@salc.org", "userNumber1@salc.org", "user_number1@salc.org"})
    void testValidEmailTest(String email) {
        assertTrue(emailValidator.test(email));
    }
    
    @ParameterizedTest
    @ValueSource(strings = {"test_admin", "test@hotmail.com", "notUser@.salc.org", "user@salc@org"})
    void testInvalidEmailTest(String email) {
        assertFalse(emailValidator.test(email));
    }
```

Este test hace uso de: 
```import org.junit.jupiter.params.ParameterizedTest;``` y de ```import org.junit.jupiter.params.provider.ValueSource;```